target/scripts/startup/fixes-stack.sh

```diff
@@ -21,7 +21,7 @@ function _fix_var_mail_permissions
   if find /var/mail -maxdepth 3 -a \( \! -user 5000 -o \! -group 5000 \) | read -r
   then
     _notify 'inf' 'Fixing /var/mail permissions'
-    chown -R 5000:5000 /var/mail || _shutdown 'Failed to fix /var/mail permissions'
+    chown -R 5000:5000 /var/mail || _notify 'err' 'Failed to fix /var/mail permissions'
```

Comment:
Not sure about this one. If the permissions cannot be fixed, normal DMS operation is questionable?

Reply:
I agree. This should remain as a fatal error if you ask me:)

Reply:
@georglauterbach this will be executed only if some process that requires writing was not successfully, and changing the owner to 5000 failed? So in this case, changing the owner is the recovering mechanism if I see correctly?

Reply:
@MohammedNoureldin I believe the container runs as the `docker` user (that is, not the host `docker` user if one exists, but the [internal one we create with the UID 5000](https://github.com/docker-mailserver/docker-mailserver/blob/e4eca00b252ce59e0e3c1e7f59ebccd11b2461c1/Dockerfile#L162)). These checks are run during startup, so the container is ensuring it owns and manages the `/var/mail` folder.

Reply:
@polarathene I am not sure if this is really important, because I used to use another directory for the email, that is why I did not really care about */var/mail*. Yes it is the default directory, but I find it again not as fatal error that deserves shutting down, especially if the user knows that he is doing it on purpose. Anyway, at least this should not happen normally and should not cause the container to crash even if the user decided to use another directory.

Reply:
> I used to use another directory to store the emails, that is why I did not really care about _/var/mail_.

This is an internal container location. The host system can volume mount any location to the containers `/var/mail`. I assume you are talking about the local host location being different from `/var/mail` for you? That's fine.

Reply:
@polarathene actually I meant the internal location in the container, but you are right, this makes not much sense to change it (though we can). Especially that it can be counted on the system somewhere else.

Reply:
I disagree as having the correct permissions on this directory is rather important. It should therefore end the container start if something went wrong when changing permissions.

Reply:
@georglauterbach I agree with you in case the user is using that directory, but this directory can be configured in Dovecot to somewhere else, and by this the user still can use a different directory with invalid permissions. But as you said, this makes sense when using the default directory, thus it can be left as fatal error at the moment.

Reply:
You can of course overwrite all the configuration of this project, but we need to assure correct functionality for _the defaults provided by us_. When you change the defaults, you'll have to make sure all works well for you.

Reply:
@georglauterbach you are right. As long as it does not prevent changing the directory and shutdown the container, this is a nice check. And it seems to only have benefits.